apps/intellij/src/main/kotlin/dev/nx/console/ide/ProjectGraphErrorProblemProvider.kt

```diff
@@ -0,0 +1,96 @@
+package dev.nx.console.ide
+
+import com.intellij.analysis.problemsView.FileProblem
+import com.intellij.analysis.problemsView.Problem
+import com.intellij.analysis.problemsView.ProblemsCollector
+import com.intellij.analysis.problemsView.ProblemsProvider
+import com.intellij.codeInsight.daemon.impl.HighlightInfo
+import com.intellij.codeInsight.daemon.impl.HighlightInfo.newHighlightInfo
+import com.intellij.codeInsight.daemon.impl.HighlightInfoType
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFileManager
+import com.intellij.workspaceModel.ide.getInstance
+import dev.nx.console.models.NxError
+import dev.nx.console.nxls.NxWorkspaceRefreshListener
+import dev.nx.console.nxls.NxlsService
+import dev.nx.console.utils.nxBasePath
+import java.io.File
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@Service(Service.Level.PROJECT)
+class ProjectGraphErrorProblemProvider(val project: Project, val cs: CoroutineScope) {
+    private val nxProblemsProvider = NxProblemsProvider(project)
```

Comment:
What is this for? I don't see any usage of `nxProblemsProvider`. It's just for `FileProblem`? Does it need the override there? A comment explaining would be helpful.

Reply:
It's just there to be passed to `FileProblem`. `override` just specifies that it's a property that exists in the interface, so it's required. 
You mean a comment above `nxProblemsProvider`? Like `this is required by FileProblem` or something alike? Just making sure I understand you correctly :)

Reply:
Because the class the empty, I thought it isn't needed, but maybe just a `// Empty class because FileProblem requires override` to explain why there's an empty class.

Reply:
Yeah I think it's mostly there to hold the project honestly... I can add a comment